pkg/database-types/mysql.go

```diff
@@ -82,6 +84,20 @@ func (m *mysql) TranslateOpenMetadataConfigToFybrikConfig(tableName string,
 	delete(ret, Username)
 	delete(ret, Password)
 
+	if databaseSchema, ok := config[DatabaseSchema]; ok {
+		delete(ret, DatabaseSchema)
+		ret[Database] = databaseSchema
+	}
+
+	if hostPort, ok := config[HostPort]; ok {
+		delete(ret, HostPort)
+		s := strings.Split(hostPort.(string), ":")
+		ret[Host] = s[0]
+		if port, err := strconv.Atoi(s[1]); err != nil {
```

Comment:
if the `hostPort` string doesn't contain port (the ":" character), you will get panic here
`panic: runtime error: index out of range [1] with length 1`